core/worker.go

```diff
@@ -950,13 +950,13 @@ func (w *worker) fillTransactions(interrupt *int32, env *environment, block *typ
 		}
 	}
 	if len(localTxs) > 0 {
-		txs := types.NewTransactionsByPriceAndNonce(env.signer, localTxs, env.header.BaseFee())
+		txs := types.NewTransactionsByPriceAndNonce(env.signer, localTxs, env.header.BaseFee(), false)
```

Comment:
Where is the sort set to true on a call to  `NewTransactionsByPriceAndNonce`?

Reply:
It's not set to true

Reply:
Then why include it as a parameter?

Reply:
@wizeguyy 

> I would prefer this was just a "sort" flag passed into NewTransactionsByPriceAndNonce() rather than a whole new function.

Reply:
but we never call it as true? do we have intention to do that later?